### PR TITLE
Fix tab issue toc

### PR DESCRIPTION
**Pull request by Arishkumar**

### CODE REVIEW

File: src/js/TreeView.jsx [hunk 1]
```diff
@@ -199,14 +199,14 @@ export class TreeNode extends React.Component {
   <li className= {'list-group-item ' + (this.state.expanded ? 'selected': '') + (this.props.currentDepth > 1 ? ' toc-child' : ' toc-parent')}
           onKeyDown={this.handleKeyDown}
           ref={list => this.list = list}>
-        <span href= "javascript:void(0)"
+        <a href= "javascript:void(0)"
```
Comment (StommePoes) [inline src/js/TreeView.jsx]: Why are we not using a button element for button behaviour?

File: src/js/TreeView.jsx [hunk 2]
```diff
@@ -199,14 +199,14 @@ export class TreeNode extends React.Component {
...
           className= {classStr}
           role= "button"
           aria-controls= {this.props.node.urn}
           aria-expanded= {(doToggle ? (this.state.expanded ? true : false) : '')}
           onClick= {((this.props.node[childField] && this.props.node[childField].length > 0 && currentDepth === 1)  ? this.toggle.bind(this) : this.handleLinkClick.bind(this, this.props.node.urn))}>
           <label className= "title">{this.props.node.title ? this.props.node.title : this.props.node.label}</label>
```
Comment (StommePoes) [inline src/js/TreeView.jsx]: Why are we using a label here? There is no input control being labelled. Simple text inside either anchors or buttons will label those anchor/button controls fine.

Comment (RamachandranMuthiah) [inline src/js/TreeView.jsx]: Thanks for your comments label is removed in next commit

File: src/js/TreeView.jsx (hunk 1 above)
Comment (RamachandranMuthiah) [inline src/js/TreeView.jsx]: Thanks currently we customised the <a> tag, yes we can use button

**Pull request merged**